src/locutus/api/table_mappings.py

```diff
@@ -40,7 +40,7 @@ def delete(cls, id):
             return ("mappings DELETE requires an editor!", 400, default_headers)
 
         table = Table.get(id)
-        mapping_count = table.terminology.dereference().delete_mappings(editor=editor)
+        mapping_count = table.terminology.dereference().soft_delete_mappings(editor=editor)

```

Comment:
I'm on the fence as to whether this has to change, but I don't think it's necessary to clarify that this delete isn't necessarily a true delete at this level. I think the implementation details are unnecessary at this level and, if we end up deciding the soft delete was a bad idea, you would have to change the function names again. Typically, I think this type of clarity is more appropriate in comments and formal API documentation rather than function names.